knative-operators.catalogsource.yaml

```diff
@@ -143,191 +108,308 @@ data:
       spec:
         group: eventing.knative.dev
         names:
           categories:
           - all
           - knative
           - eventing
-          kind: ClusterProvisioner
-          plural: clusterprovisioners
-          singular: clusterprovisioner
-        scope: Cluster
-        version: v1alpha1
-    - apiVersion: apiextensions.k8s.io/v1beta1
-      kind: CustomResourceDefinition
-      metadata:
-        name: eventsources.feeds.knative.dev
-      spec:
-        group: feeds.knative.dev
-        names:
-          categories:
-          - all
-          - knative
-          - keventsource
-          - eventing
-          kind: EventSource
-          plural: eventsources
-          singular: eventsource
-        scope: Namespaced
-        version: v1alpha1
-    - apiVersion: apiextensions.k8s.io/v1beta1
-      kind: CustomResourceDefinition
-      metadata:
-        name: eventtypes.feeds.knative.dev
-      spec:
-        group: feeds.knative.dev
-        names:
-          categories:
-          - all
-          - knative
-          - keventtype
-          - eventing
-          kind: EventType
-          plural: eventtypes
-          singular: eventtype
-        scope: Namespaced
-        version: v1alpha1
-    - apiVersion: apiextensions.k8s.io/v1beta1
-      kind: CustomResourceDefinition
-      metadata:
-        name: feeds.feeds.knative.dev
-      spec:
-        group: feeds.knative.dev
-        names:
-          categories:
-          - all
-          - knative
-          - eventing
-          kind: Feed
-          plural: feeds
-          singular: feed
-        scope: Namespaced
-        version: v1alpha1
-    - apiVersion: apiextensions.k8s.io/v1beta1
-      kind: CustomResourceDefinition
-      metadata:
-        name: flows.flows.knative.dev
-      spec:
-        group: flows.knative.dev
-        names:
-          categories:
-          - all
-          - knative
-          - eventing
-          kind: Flow
-          plural: flows
-          singular: flow
+          kind: Subscription
+          plural: subscriptions
+          shortNames:
+          - sub
+          singular: subscription
         scope: Namespaced
         version: v1alpha1
     - apiVersion: apiextensions.k8s.io/v1beta1
       kind: CustomResourceDefinition
       metadata:
-        name: sources.eventing.knative.dev
+        creationTimestamp: null
+        labels:
+          controller-tools.k8s.io: "1.0"
+        name: containersources.sources.eventing.knative.dev
       spec:
-        group: eventing.knative.dev
+        group: sources.eventing.knative.dev
         names:
           categories:
           - all
           - knative
           - eventing
-          kind: Source
-          plural: sources
-          singular: source
+          - sources
+          kind: ContainerSource
+          plural: containersources
         scope: Namespaced
+        validation:
+          openAPIV3Schema:
+            properties:
+            - apiVersion:
```

Comment:
These properties should be a map, not an array. If you compare this with the values in `etc/upstream/knative-eventing-sources-0.2.0.yaml` you'll see what I mean. Somehow this got munged in this file only.